### DISCUSSION TITLE
Webhook for WalletConnect v2 Session Requests

### DISCUSSION THREAD

**Discussion by PaDuckk** [WalletConnect/walletconnect-monorepo#2525] [Developer Support]
I am developing a web extension wallet that supports WalletConnect, is there any way to get a webhook to my server for sessionRequest in WalletConnect v2?

**Comment by vivekkumar15** [comment]
We have a similar use case, is there a way to get the messages to the backend server through webhook?

**Comment by nsaun-dq** [comment]
I needed a session delete webhook for a similar usecase. any updates on this till now? @PaDuckk @vivekkumar15
